### PR TITLE
Feature/211 search documentation

### PR DESCRIPTION
**Pull request by naveen2507**
Add documentation on the search sidebar with several example queries. e.g.

- [x] search for a word
- [x] search for multiple words
- [x] search for POS tag
- [x] search for lemma + POS
- [x] search for a particular kind of named entity (e.g. a PERSON)
- [x] search for a particular concept
- [x] include reference to MTAS documentation

### CODE REVIEW

File: inception-search-mtas/src/main/resources/META-INF/asciidoc/user-guide/search-mtas.adoc [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+ [Lemma.Lemma="be"][Lemma.Lemma="sign"]
+
+* Sequence: all occurrences of the token *house* immediately followed by a verb
+
+ [Token=“house"][POS.PosValue=“VERB"]
+
+* Sequence: all occurrences of a verb immediately followed by a named entity
+
+ [POS.PosValue=“VERB"]<Named_entity/>
```
[the remark this answers is not in the document]
Comment (betoboullosa) [reply inception-search-mtas/src/main/resources/META-INF/asciidoc/user-guide/search-mtas.adoc]: ok

File: inception-search-mtas/src/main/resources/META-INF/asciidoc/user-guide/search-mtas.adoc [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+ The capital of Galicia
+ 
+* Lemma: all occurrences of the lemma *sign*
+
+ [Lemma.Lemma="sign"]
+
+* POS tag: all tokens that are annotated as a noun
+
+ [POS.PosValue="NOUN"]
```
[the remark this answers is not in the document]
Comment (betoboullosa) [reply inception-search-mtas/src/main/resources/META-INF/asciidoc/user-guide/search-mtas.adoc]: Added an explanation about it

File: inception-search-mtas/src/main/resources/META-INF/asciidoc/user-guide/search-mtas.adoc [hunk 3]
```diff
@@ -0,0 +1,138 @@
+[[sect_search-mtas]]
+
+==== Mtas search syntax
+
+The INCEpTION Mtas search provider allows queries to be executed using CQL (Corpus 
+Query Language), as shown in the following examples.
+More examples and information about CQL syntax can be found 
+in https://meertensinstituut.github.io/mtas/search_cql.html.
+
```
[the remark this answers is not in the document]
Comment (betoboullosa) [reply inception-search-mtas/src/main/resources/META-INF/asciidoc/user-guide/search-mtas.adoc]: Done

File: inception-search-mtas/src/main/resources/META-INF/asciidoc/user-guide/search-mtas.adoc [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+* Generic Search over annotated KB entities : all occurrences for KB entity *Bordeaux*
+
+ <KB.Entity="Bordeaux"/>
+
+* Named Entity Identifier : all occurrences of the annotated identifier *ChateauMorgonBeaujolais*
+	
+ [Named_entity.identifier="ChateauMorgonBeaujolais"]
+
+ <Named_entity.identifier="ChateauMorgonBeaujolais"/>
```
[the remark this answers is not in the document]
Comment (naveen2507) [reply inception-search-mtas/src/main/resources/META-INF/asciidoc/user-guide/search-mtas.adoc]: Done

**Comment by reckart** [comment]
@betoboullosa please merge once the build is good.

**Pull request merged**